src/main/webapp/discussion.js

```diff
@@ -160,15 +121,13 @@ function createComment(comment) {
   }
   repliesDiv.appendChild(repliesList);
 
-  if (AUTH_STATUS.loggedIn) {
```

Comment:
Why do you no longer check if the user is logged in?

Reply:
Because this page (view.html) can only be viewed by logged in users (thanks to the security policy in web.xml)